Address ConsoleApp/Address. ConsoleApp.csproj

```diff
@@ -0,0 +1,9 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Till it is the same
<img width="275" alt="image" src="https://user-images.githubusercontent.com/89320816/144274052-755e4ecf-a763-40c1-b372-d6765fe8211e.png">